Raiffeisen Mein Elba: Improve parsing of Payee and Bank Account

akirk opened the pull request:
This extends the parsing so that it now produces proper info for the payee, the bank account and the memo (just the Verwendungszweck).

akirk committed:
Update encoding of Raiffeisen files

akirk committed:
Improve parsing of payee and bank accounts

nblock commented:
Thx for your contribution! Just two nitpicks:

* Due to your change, the unittests fail
* Could you provide a (anonymized and minimal) CSV export as it is currently generated by Raiffeisen?

nblock commented:
Could you please provide a minimal and anonymized CSV export from Raiffeisen that can be included to fix/update the tests?

akirk committed:
test data

akirk committed:
Fix unit tests

akirk commented:
@nblock sorry for the long wait, I've now added tests and fixed the existing ones.

I didn't know how to run unit tests, so here are some instructions of what I did:

I cloned https://github.com/kedder/ofxstatement and [followed the test instructions there](https://github.com/kedder/ofxstatement#development--test) and then `ln -s`ed the `plugins` directory from this repo and ran `pytest` from within the environment.

akirk committed:
Add BOM to test file

JoelColledge commented:
I just used this. It works well :+1: I am for merging it.
